src/NovakSolutions/Infusionsoft/Model/OrderShipping.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: joey
+ * Date: 7/23/18
+ * Time: 7:20 PM
+ */
+
+namespace NovakSolutions\Infusionsoft\Model;
+use NovakSolutions\Infusionsoft\Enum\FieldTypes;
+use NovakSolutions\Infusionsoft\Service\ContactService;
+
+/**
+ * Class Contact
+ * @package NovakSolutions\Infusionsoft\Model
+ * @property Address[] addresses
+ * @property \DateTime birthday
+ * @property EmailAddress[] email_addresses
+ * @property int id
+ */
+class OrderShipping extends Model
+{
+    protected static $fields = [
+        'company' => FieldTypes::STRING,
+        'country_code' => FieldTypes::STRING,
```

Comment:
FYI, the automation center now has a country name to code tool, if you ever need a small lib that does it there is an example.